shared/js/dataDriver.js

```diff
@@ -75,6 +75,8 @@ loop.DataDriver = function() {
         throw new Error("Missing option dispatcher");
       }
 
+      this._retryTimeout = this.INITIAL_RETRY_TIMEOUT;
```

Comment:
Yes. This is true. But it seemed weird to have a member that isn't defined until the connection is opened. Maybe I should set this to null here instead.